SWAPI: Flag alpha fits with out-of-range proton-to-alpha peak ratios (#168)
IMAP-Science-Operations-Center/imap_L3_processing: docs/swapi/alpha-sw.md

Some notes on error handling:
- If the proton fit fails, fill values are reported and the proton quality flag is propagated.
- If the alpha fit fails at any point, fill values are reported and `FIT_ERROR` is set.

> - If the fitted alpha bulk speed is less than 90% of the fitted proton bulk speed, fill values are reported and `BAD_FIT` is set. This guards against the proton shoulder being confused for the alpha population.

Reviewer: ```suggestion
- If the fitted alpha bulk speed is less than 90% of the fitted proton bulk speed, fill values are reported and `BAD_FIT` is set. This is intended to avoid confusing the proton beam population for the alpha population.
```
